app/src/main/java/com/epam/themes/compoundview/DrawerFragment.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.themes.compoundview;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import com.epam.cleancodetest.R;
+
+public class DrawerFragment extends Fragment {
+    private String pageNumber;
+
+    public static DrawerFragment newInstance(int pageNum) {
+        DrawerFragment fragment = new DrawerFragment();
+        fragment.pageNumber = String.valueOf(pageNum);
```

Comment:
It's wrong put something in fragment field in constructor. 
Please add variable to bundle, put it into fragment and in onCreate method read value from bundle

Reply:
HW 11.03.  Fixed